plugins/deploy_env_vars/test/samson_deploy_env_vars/samson_plugin_test.rb

```diff
@@ -18,15 +18,13 @@
   end
 
   describe :deploy_env do
-    it "returns an empty hash" do
-      Samson::Hooks.fire(:deploy_env, deploy).must_equal([{}])
```

Comment:
simpler to use `only_callbacks_for_plugin` so other plugins don't break this test again